src/networkextension.cs

```diff
@@ -1321,6 +1322,7 @@ interface NEPacketTunnelNetworkSettings {
 	[BaseType (typeof(NSObject))]
 	interface NEPacketTunnelFlow {
 		[Export ("readPacketsWithCompletionHandler:")]
+		[Async (ResultType = typeof (NEPacketTunnelFlowReadResult))]
```

Comment:
The discussion seems fine: _Each call to this method results in a single execution of the completion handler. The caller should call this method after each completionHandler execution in order to continue to receive packets from the TUN interface._